nixos/modules/hardware/saleae-logic.nix

```diff
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.saleae-logic;
+in
+{
+  options.hardware.saleae-logic = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = ''
+        Enables udev rules for Saleae Logic devices.
+      '';
+    };
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.saleae-logic-2;
+      defaultText = lib.literalExpression "pkgs.saleae-logic-2";
+      description = ''
+        Saleae Logic package to use.
```

Comment:
This should probably mention you can also add version 1 here?

Right now, I'm using a cheap logic analyzer that only works with version 1, but I could assume I'd buy a new one (and then would want to use version 2).

We could probably even make this a list, so both can be added?

Reply:
To be honest, it doesn't really matter. Both packages contain the same udev rules, so if your analyzer only supports Logic 1.x it should still work file.

Reply:
Indeed!